formspree/routes.py

```diff
@@ -30,6 +30,8 @@ def configure_routes(app):
     app.add_url_rule('/forms', 'forms', view_func=forms.views.forms, methods=['GET', 'POST'])
     app.add_url_rule('/forms/<hashid>/', 'form-submissions', view_func=forms.views.form_submissions, methods=['GET'])
     app.add_url_rule('/forms/<hashid>.<format>', 'form-submissions', view_func=forms.views.form_submissions, methods=['GET'])
+    app.add_url_rule('/forms/<hashid>/delete', 'form-deletion', view_func=forms.views.form_deletion, methods=['GET'])
+    app.add_url_rule('/forms/<hashid>/<submissionid>/delete', 'submission-deletion', view_func=forms.views.submission_deletion, methods=['GET'])
```

Comment:
Even with the authentication methods below, should we have a confirmation screen just for the users sake? I think we should for form deletion, but if we do for submission deletion do you think it becomes too cumbersome?